javascript-sdk/examples/list-query-usage/list-query-usage.js

```diff
@@ -0,0 +1,37 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+const path = require('path');
+const fs = require('fs');
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  configure({ apiKey: process.env.AGENTQL_API_KEY });
```

Comment:
I recommend moving the keys to the installation docs, not the example files

Reply:
Removed it from the example.